README.md

```diff
@@ -9,13 +9,13 @@ Atlassian JWT Authentication provides support for handling JWT authentication as
 
 You can check out the latest source from git:
 
-`git clone https://github.com/MeisterLabs/atlassian-jwt-authentication.git`
+`git clone https://github.com/HeroCoders/atlassian-jwt-authentication.git`
 
 Or, if you're using Bundler, just add the following to your Gemfile:
 
 ```ruby
 gem 'atlassian-jwt-authentication', 
-  git: 'https://github.com/MeisterLabs/atlassian-jwt-authentication.git'
+  git: 'https://github.com/HeroCoders/atlassian-jwt-authentication.git'
```

Comment:
These two changes here are unimportant. Only relevant ones are the renames of those two files below (gemspec and lib/a-j-a.rb)